<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaffoldRigidAlignerWidget</class>
 <widget class="QWidget" name="ScaffoldRigidAlignerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1957</width>
    <height>998</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QDockWidget" name="dockWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>410</width>
       <height>135</height>
      </size>
     </property>
     <property name="features">
      <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
     </property>
     <property name="allowedAreas">
      <set>Qt::AllDockWidgetAreas</set>
     </property>
     <property name="windowTitle">
      <string>Scaffold Rigid Aligner</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>408</width>
            <height>956</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="QToolBox" name="toolBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolBox::tab {
         background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                     stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
         border-radius: 5px;
         color: black;
     }

     QToolBox::tab:selected { /* italicize selected tabs */
         font: bold;
         color: black;
     }
QToolBox {
    padding : 0
}</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <property name="tabSpacing">
              <number>6</number>
             </property>
             <widget class="QWidget" name="alignPage">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>404</width>
                <height>892</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <attribute name="label">
               <string>Initial Rigid Alignment</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>7</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>3</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>3</number>
               </property>
               <item>
                <widget class="QGroupBox" name="alignSettingsGroupBox">
                 <property name="title">
                  <string>Scaffold alignment</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <property name="spacing">
                   <number>3</number>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                  <item>
                   <widget class="QFrame" name="swapAxis_frame">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>59</height>
                     </size>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_6">
                     <item row="1" column="5">
                      <widget class="QRadioButton" name="scaffoldX_radioButton">
                       <property name="text">
                        <string>X</string>
                       </property>
                      </widget>
                     </item>
                     <item row="6" column="0" colspan="2">
                      <widget class="QLabel" name="rotateAxis_label">
                       <property name="text">
                        <string>Euler orientation:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="scaffoldAxis_label">
                       <property name="text">
                        <string>Scaffold UP axis:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="4">
                      <widget class="QRadioButton" name="scaffoldY_radioButton">
                       <property name="text">
                        <string>Y</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="3">
                      <widget class="QRadioButton" name="scaffoldZ_radioButton">
                       <property name="text">
                        <string>Z</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="5">
                      <widget class="QRadioButton" name="dataX_radioButton">
                       <property name="text">
                        <string>X</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="3">
                      <widget class="QRadioButton" name="dataZ_radioButton">
                       <property name="text">
                        <string>Z</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="4">
                      <widget class="QRadioButton" name="dataY_radioButton">
                       <property name="text">
                        <string>Y</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0">
                      <widget class="QLabel" name="dataAxis_label">
                       <property name="text">
                        <string>Data UP axis:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="4" column="0">
                      <widget class="QLabel" name="upsideDown_label">
                       <property name="text">
                        <string>Is UP axis upside down?</string>
                       </property>
                      </widget>
                     </item>
                     <item row="7" column="3">
                      <widget class="QDoubleSpinBox" name="yaw_doubleSpinBox"/>
                     </item>
                     <item row="6" column="3">
                      <widget class="QLabel" name="yaw_label">
                       <property name="text">
                        <string>Yaw</string>
                       </property>
                      </widget>
                     </item>
                     <item row="7" column="4">
                      <widget class="QDoubleSpinBox" name="pitch_doubleSpinBox"/>
                     </item>
                     <item row="6" column="4">
                      <widget class="QLabel" name="pitch_label">
                       <property name="text">
                        <string>Pitch</string>
                       </property>
                      </widget>
                     </item>
                     <item row="7" column="5">
                      <widget class="QDoubleSpinBox" name="roll_doubleSpinBox"/>
                     </item>
                     <item row="6" column="5">
                      <widget class="QLabel" name="roll_label">
                       <property name="text">
                        <string>Roll</string>
                       </property>
                      </widget>
                     </item>
                     <item row="4" column="3">
                      <widget class="QCheckBox" name="upsideDown_checkBox">
                       <property name="text">
                        <string>Flip</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="alignLoadSaveWidgets" native="true">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <layout class="QGridLayout" name="gridLayout" rowstretch="0">
                     <property name="margin">
                      <number>0</number>
                     </property>
                     <property name="spacing">
                      <number>3</number>
                     </property>
                     <item row="0" column="0">
                      <widget class="QPushButton" name="saveSettingsButton">
                       <property name="toolTip">
                        <string>Offset the model to the centre of the data points. May need to click View All afterwards.</string>
                       </property>
                       <property name="text">
                        <string>Save Settings</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QPushButton" name="loadSettingsButton">
                       <property name="enabled">
                        <bool>true</bool>
                       </property>
                       <property name="toolTip">
                        <string>Load pre-saved alignment settings</string>
                       </property>
                       <property name="text">
                        <string>Load Settings</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>10</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="alignResetButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="toolTip">
                  <string>Reset the alignment settings</string>
                 </property>
                 <property name="text">
                  <string>Reset</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="margin">
               <number>3</number>
              </property>
              <item>
               <widget class="QPushButton" name="viewAllButton">
                <property name="toolTip">
                 <string>Adjust the view to see the whole model</string>
                </property>
                <property name="text">
                 <string>View All</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="doneButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="toolTip">
                 <string>Finish this step</string>
                </property>
                <property name="text">
                 <string>Done</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="BaseSceneviewerWidget" name="sceneviewerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BaseSceneviewerWidget" name="overlaySceneviewerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>750</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1000</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseSceneviewerWidget</class>
   <extends>QWidget</extends>
   <header>opencmiss/zincwidgets/basesceneviewerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
